declarative-examples/buildPluginParentPOM.groovy

```diff
@@ -0,0 +1,29 @@
+pipeline {
+    tools {
+        maven "apache-maven-3.1.0"
+        jdk "default"
+    }
+
+    agent {
+        label "master"
```

Comment:
If i used `agent any` that was a mistake. I'll remove any reference to `agent any`